src/sentry/api/serializers/models/userreport.py

```diff
@@ -12,34 +12,47 @@ def get_attrs(self, item_list, user):
         queryset = list(EventUser.objects.filter(
             id__in=[i.event_user_id for i in item_list],
         ))
+
         event_users = {e.id: d for e, d in zip(queryset, serialize(queryset, user))}
 
+        # If a event list with multiple project IDs is passed to this and event IDs are not unique
+        # this could return the wrong eventIDs
+        events_list = Event.objects.filter(
+            project_id__in={i.project_id for i in item_list},
+            event_id__in=[i.event_id for i in item_list]
+        ).values('id', 'event_id')
+
+        events_dict = {e['event_id']: e['id'] for e in events_list}
+
         attrs = {}
         for item in item_list:
             attrs[item] = {
                 'event_user': event_users.get(item.event_user_id),
+                'event_id': events_dict.get(item.event_id)
```

Comment:
this is the database id, right? if so, it think it'd be less confusing if you called it `id` or did something like `'event': {'id': events_dict.get(item.event_id)}` since `event_id` is mostly used here to refer to the uuid

Reply:
`event_id` here is also the event.event_id. It's the UUID.

Reply:
`item.event_id` is the uuid, but `events_dict.get(item.event_id)` should return the database id/pk right? so i think it's confusing that the key would be `event_id`

Reply:
Oh, you're right. You're smarter than me. 🎒